scripts/create_x509.go

```diff
@@ -0,0 +1,88 @@
+package main
```

Comment:
one suggestion: maybe it worth to add this thing to the Makefile and then mention in the documentation that one can build it and use the binary to generate certificates?

Reply:
Good call, I actually had such Makefile section during the development 😅